src/terms/Term.cpp

```diff
@@ -109,5 +109,7 @@ namespace knowrob::py {
 				.def("isBlank", &Term::isBlank)
 				.def("isGround", &Term::isGround)
 				.def("variables", pure_virtual(&Term::variables), return_value_policy<copy_const_reference>());
+		// register shared_ptr to Term
+		register_ptr_to_python< std::shared_ptr< Term > >();
```

Comment:
I think this is not needed. the pointer is registered in the `class_` statement or not?

Reply:
I tested it again without this line, but then the kb_positive_query test fails in line 136 when getting the substitution.